src/server/options.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Options for a `manticore` "server" and "client" for a RoT.
+
+use crate::protocol::capabilities;
+use crate::protocol::device_id;
+
+/// Options struct for initialising a [`crate::server::pa_rot::PaRot`]
```

Comment:
Doc nit: I'm not a fan of long names in documentation. Can you shorten them to PaRot and AcRot, and `use` these names under a `#[cfg(doc)]`?